cmake/CMakeLists.txt

```diff
@@ -538,7 +538,10 @@ set(CMAKE_TUNE_FLAGS "${CMAKE_TUNE_DEFAULT}" CACHE STRING "Compiler and machine
 separate_arguments(CMAKE_TUNE_FLAGS)
 foreach(_FLAG ${CMAKE_TUNE_FLAGS})
   target_compile_options(lammps PRIVATE ${_FLAG})
-  target_compile_options(lmp PRIVATE ${_FLAG})
+  # skip these flags when linking the main executable
```

Comment:
Because they cause errors on multiple of my development machines and the flags were only added to suppress warnings when compiling non-kokkos non-GPU code through nvcc with nvcc_wrapper.

Reply:
Makes sense, thanks for the explanation! It would be good to mention that in the comments as well then.